Tools/autotest/pysim/util.py

```diff
@@ -230,8 +237,15 @@ def valgrind_log_filepath(binary, model):
 
 
 def kill_screen_gdb():
-    cmd = ["screen", "-X", "-S", "ardupilot-gdb", "quit"]
-    subprocess.Popen(cmd)
+    global windowID
+    if sys.platform == "darwin":
```

Comment:
I'm not really sure why we're closing windows here.  We use screen when we *don't* have a windowing system

Reply:
For mac os I made a change to start debugger on a separate terminal.

Reply:
right I should check for DISPLAY here.